docs/articles/documentation/using-testcafe/common-concepts/concurrent-test-execution.md

```diff
@@ -47,6 +47,8 @@ testcafe -c 4 safari,firefox tests/test.js
 
 In this case, tests will be distributed across four Safari instances and the same tests will also run in four Firefox instances.
```

Comment:
In this case, tests are distributed across four Safari instances and the same tests are also run in four Firefox instances.